CONTRIBUTING.md

```diff
@@ -18,6 +18,25 @@ reason about the proposed changes.
 ## Running the tests
 
 Many of the tests require privileges to set resource limits and load eBPF code.
-The easiest way to obtain these is to run the tests with `sudo`:
+The easiest way to obtain these is to run the tests with `sudo`.
+
+To test the current package with your local kernel you can simply run:
+```bash
+sudo go test ./...
```

Comment:
This should be replaced with `go test -exec sudo ./...`. Unlike `sudo go test ./...`, it won't execute the Go build as sudo, only the resulting `a.out` artifacts.